src/main/java/net/atos/zac/app/zaken/converter/RESTZaaktypeConverter.java

```diff
@@ -5,8 +5,8 @@
 
 package net.atos.zac.app.zaken.converter;
 
-import static net.atos.client.zgw.ztc.util.ZaakTypeUtil.isNuGeldig;
-import static net.atos.client.zgw.ztc.util.ZaakTypeUtil.isServicenormBeschikbaar;
+import static net.atos.client.zgw.ztc.util.ZaakTypeUtilKt.isNuGeldig;
```

Comment:
Not sure I like that it has been renamed, but ok...

Reply:
I don't think I can avoid that. Well, I could if I make ZaakTypeUtil a class or object but I think that is not idiomatic Kotlin. This file just contains static functions so I see no need for a class or object right?